app/controllers/reviews_controller.rb

```diff
@@ -1,2 +1,12 @@
 class ReviewsController < ApplicationController
+  def create
+    @champion = Champion.find(params[:champion_id])
+    @review = Review.new(review_params)
+    @review.champion = @champion
+    if @review.save
+      redirect_to champion_path(@champion)
+    else
+      render "champions/show"
```

Comment:
I don't think we have a show page for champion. Do you think we should have one?

Reply:
How can you see the details for the champion? and how can you see the reviews of the champion?